dev/tool/src/restoreGithub.ts

```diff
@@ -0,0 +1,80 @@
+import { type IntegrationKind, type PersonId, type WorkspaceUuid, systemAccountUuid } from '@hcengineering/core'
+import { getAccountClient } from '@hcengineering/server-client'
+import { generateToken } from '@hcengineering/server-token'
+import { type Integration } from '@hcengineering/account-client'
+import { getDBClient } from '@hcengineering/postgres'
+
+const GITHUB_INTEGRATION: IntegrationKind = 'github' as any
+
+export async function restoreGithubIntegrations (dbUrl: string, dryrun: boolean): Promise<void> {
+  try {
+    const pg = getDBClient(dbUrl)
+    const pgClient = await pg.getClient()
+    const token = generateToken(systemAccountUuid, undefined, { service: 'admin', admin: 'true' })
+    const accountClient = getAccountClient(token)
+
+    const integrationSettings = await pgClient<
```

Comment:
It could be few of them In workspace, you need to group by account and pass an array as installationId

Reply:
Thanks, fixed